### PR TITLE
6 refactor packages and class names

### PR DESCRIPTION
**Pull request by Alex-Schiff**
Thank you for taking time to contribute this pull request :smile:

You have probably already checked the following points, but as a reminder, please ensure that:

* your branch is rebased on the latest version of the master branch
* all tests pass
* code is formatted with IntelliJ Idea default settings (if you use another IDE, never mind, ignore this point)

You may add the reference of the issue in your commit message.

Don't hesitate to add your name to the contributors list in the `README.md` file.

Many thanks upfront!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,21 +169,21 @@ public void testPersistPerson() throws Exception {
 }
 ```
 
-There are many other uses cases where Easy Random can be useful, you can find a non exhaustive list in the [wiki](https://github.com/j-easy/easy-random/wiki/use-cases).
+There are many other uses cases where Easy Random can be useful, you can find a non exhaustive list in the [wiki](https://github.com/k-random/k-random/wiki/use-cases).
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -169,21 +169,21 @@ public void testPersistPerson() throws Exception {
...
 
 ## Extensions
 
 * [JUnit extension](https://glytching.github.io/junit-extensions/randomBeans): Use Easy Random to generate random data in JUnit tests (courtesy of [glytching](https://github.com/glytching))
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ person.setGender(Gender.MALE);
 person.setAddress(address);
 ```
 
-**With** Easy Random, generating a random `Person` object is done with `new EasyRandom().nextObject(Person.class)`.
+**With** Easy Random, generating a random `Person` object is done with `new KRandom().nextObject(Person.class)`.
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -64,40 +64,40 @@ EasyRandomParameters parameters = new EasyRandomParameters()
...
 # Why Easy Random ?
 
 Populating a Java object with random data can look easy at first glance, unless your domain model involves many related classes. In the previous example, let's suppose the `Person` type is defined as follows:
 
 <p align="center">
-    <img src="https://raw.githubusercontent.com/wiki/j-easy/easy-random/images/person.png">
+    <img src="https://raw.githubusercontent.com/wiki/k-random/k-random/images/person.png">
 </p>
 
 **Without** Easy Random, you would write the following code in order to create an instance of the `Person` class:
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 5]
```diff
@@ -64,40 +64,40 @@ EasyRandomParameters parameters = new EasyRandomParameters()
...
 
-The static methods `named`, `ofType` and `inClass` are defined in [`org.jeasy.random.FieldPredicates`](https://github.com/j-easy/easy-random/blob/master/easy-random-core/src/main/java/org/jeasy/random/FieldPredicates.java) 
+The static methods `named`, `ofType` and `inClass` are defined in [`random.k.io.github.FieldPredicates`](https://github.com/k-random/k-random/blob/master/k-random-core/src/main/java/org/jeasy/random/FieldPredicates.java) 
 which provides common predicates you can use in combination to define exactly which fields to exclude.
-A similar class called [`TypePredicates`](https://github.com/j-easy/easy-random/blob/master/easy-random-core/src/main/java/org/jeasy/random/TypePredicates.java) can be used to define which types to exclude from the object graph.
+A similar class called [`TypePredicates`](https://github.com/k-random/k-random/blob/master/k-random-core/src/main/java/org/jeasy/random/TypePredicates.java) can be used to define which types to exclude from the object graph.
 You can of course use your own `java.util.function.Predicate` in combination with those predefined predicates. 
 
 # Why Easy Random ?
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 6]
```diff
@@ -20,38 +20,38 @@
...
-The method `EasyRandom#nextObject` is able to generate random instances of any given type.
+The method `KRandom#nextObject` is able to generate random instances of any given type.
 
-# What is this EasyRandom API ?
+# What is this KRandom API ?
 
 The `java.util.Random` API provides 7 methods to generate random data: `nextInt()`, `nextLong()`, `nextDouble()`, `nextFloat()`, `nextBytes()`, `nextBoolean()` and `nextGaussian()`.
 What if you need to generate a random `String`? Or say a random instance of your domain object?
-Easy Random provides the `EasyRandom` API that extends `java.util.Random` with a method called `nextObject(Class type)`.
+Easy Random provides the `KRandom` API that extends `java.util.Random` with a method called `nextObject(Class type)`.
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 7]
```diff
@@ -64,40 +64,40 @@ EasyRandomParameters parameters = new EasyRandomParameters()
...
+For more details about these parameters, please refer to the [configuration parameters](https://github.com/k-random/k-random/wiki/Randomization-parameters) section.
 
-In most cases, default options are enough and you can use the default constructor of `EasyRandom`.
+In most cases, default options are enough and you can use the default constructor of `KRandom`.
 
-Easy Random allows you to control how to generate random data through the [`org.jeasy.random.api.Randomizer`](https://github.com/j-easy/easy-random/blob/master/easy-random-core/src/main/java/org/jeasy/random/api/Randomizer.java) interface and makes it easy to exclude some fields from the object graph using a `java.util.function.Predicate`:
+Easy Random allows you to control how to generate random data through the [`api.io.github.krandomizer`](https://github.com/k-random/k-random/blob/master/k-random-core/src/main/java/org/jeasy/random/api/Randomizer.java) interface and makes it easy to exclude some fields from the object graph using a `java.util.function.Predicate`:
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 8]
```diff
@@ -64,40 +64,40 @@ EasyRandomParameters parameters = new EasyRandomParameters()
...
    .randomize(String.class, () -> "foo")
    .excludeField(named("age").and(ofType(Integer.class)).and(inClass(Person.class)));
 
-EasyRandom easyRandom = new EasyRandom(parameters);
-Person person = easyRandom.nextObject(Person.class);
+KRandom kRandom = new KRandom(parameters);
+Person person = kRandom.nextObject(Person.class);
 ```
 
 In the previous example, Easy Random will:
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 9]
```diff
@@ -20,38 +20,38 @@
...
 for now. Please consider upgrading to one of these versions at your earliest convenience.
 
 ## Latest news
 
-* 15/11/2020: Easy Random v5.0.0 is out and is now based on Java 11. Feature wise, this release is the same as v4.3.0. Please check the [release notes](https://github.com/j-easy/easy-random/releases/tag/easy-random-5.0.0) for more details.
-* 07/11/2020: Easy Random v4.3.0 is now released with support for generic types and fluent setters! You can find all details in the [change log](https://github.com/j-easy/easy-random/releases/tag/easy-random-4.3.0).
+* 15/11/2020: Easy Random v5.0.0 is out and is now based on Java 11. Feature wise, this release is the same as v4.3.0. Please check the [release notes](https://github.com/k-random/k-random/releases/tag/k-random-5.0.0) for more details.
+* 07/11/2020: Easy Random v4.3.0 is now released with support for generic types and fluent setters! You can find all details in the [change log](https://github.com/k-random/k-random/releases/tag/k-random-4.3.0).
 
 # What is Easy Random ?
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 10]
```diff
@@ -20,38 +20,38 @@
...
 Version 5.0.x (based on Java 11) and version 4.3.x (based on Java 8) are the only supported versions
 for now. Please consider upgrading to one of these versions at your earliest convenience.
 
 ## Latest news
 
-* 15/11/2020: Easy Random v5.0.0 is out and is now based on Java 11. Feature wise, this release is the same as v4.3.0. Please check the [release notes](https://github.com/j-easy/easy-random/releases/tag/easy-random-5.0.0) for more details.
-* 07/11/2020: Easy Random v4.3.0 is now released with support for generic types and fluent setters! You can find all details in the [change log](https://github.com/j-easy/easy-random/releases/tag/easy-random-4.3.0).
+* 15/11/2020: Easy Random v5.0.0 is out and is now based on Java 11. Feature wise, this release is the same as v4.3.0. Please check the [release notes](https://github.com/k-random/k-random/releases/tag/k-random-5.0.0) for more details.
+* 07/11/2020: Easy Random v4.3.0 is now released with support for generic types and fluent setters! You can find all details in the [change log](https://github.com/k-random/k-random/releases/tag/k-random-4.3.0).
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: Fixed.

File: README.md [hunk 11]
```diff
@@ -20,38 +20,38 @@
 ## Project status
 
 As of November 15, 2020, Easy Random is in maintenance mode. This means only bug fixes will be addressed from now on (except
```
[the remark this answers is not in the document]
Comment (Alex-Schiff) [reply README.md]: This will be address in #14.

**Review by jamalkhaffaf — APPROVED** [review]

**Pull request merged**